db/migrate/20161030141745_add_user_info_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserInfoToUser < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :username, :string
```

Comment:
If you're going to rely on a user having a username, it's best to have `null: false` on this column.